deps/thpool/thpool.c

```diff
@@ -232,21 +238,23 @@ static void redisearch_thpool_verify_init(struct redisearch_thpool_t *thpool_p)
   redisearch_thpool_lock(thpool_p);
   size_t curr_num_threads_alive = thpool_p->num_threads_alive;
   size_t n_threads = thpool_p->n_threads;
-  size_t n_new_threads = 0;
+  size_t n_new_threads, n_threads_to_revive, n_threads_to_kill;
```

Comment:
nitpick: the last two are only used inside the `if` block. Consider declaring right before using them (before the inner `if`)

Reply:
:( you said to declare them all in one line